catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -68,10 +77,31 @@
 
     private boolean enableDelete = true;
 
+    private boolean initialized = false;
+
+    private ScheduledExecutorService executor;
+
+    private int taskWaitTimeSeconds = 30;
```

Comment:
Those variables are only used/accessed in the same one thread. Is there somewhere specific that you see the use of these variables in another threads context?